src/mon.c

```diff
@@ -2151,13 +2151,17 @@ register struct monst *mtmp;
         struct obj* list = collect_all_transient(NULL);
         struct obj* next;
         if (list) {
-            if (Blind && !Deaf)
-                You("hear a rush of objects jumble together nearby.");
-            else if (Blind && Deaf)
-                You_feel("a jarring vibration nearby.");
-            else
+            if (!Deaf) {
+                pline("A voice echoes in the arena:");
+                verbalize("You may choose one trophy from the spoils!");
+                if (Blind)
```

Comment:
This really ought to keep the structure it had previously; I would prefer the announcer voice be the only thing in the !Deaf block, then the three cases for (!Blind), (Blind && !Deaf), and (Blind && Deaf) as they were. As it is here, the structure is confusing.